src/main/kotlin/uk/gov/dluhc/printapi/dto/GenerateAnonymousElectorDocumentDto.kt

```diff
@@ -0,0 +1,19 @@
+package uk.gov.dluhc.printapi.dto
+
+data class GenerateAnonymousElectorDocumentDto(
+    val gssCode: String,
+    val sourceType: SourceType,
+    val sourceReference: String,
+    val applicationReference: String,
+    val electoralRollNumber: String,
+    val photoLocation: String,
+    val certificateLanguage: CertificateLanguage,
+    val supportingInformationFormat: SupportingInformationFormat,
```

Comment:
Minor, @Neil-Massey. 

I think it will be good to rename the `supportingInfoFormat` field of the `GenerateAnonymousElectorDocumentRequest` in the open api specs as `supportingInformationFormat` as well so the mapping would be more convenient and it will be consistent with the naming of the other fields for the `supportingInformationFormat`.